src/hooks.tsx

```diff
@@ -34,9 +35,17 @@ const createReducer = <T extends DataType>() => (
         );
       }
 
+      if (state.paginationEnabled === true) {
+        rows = getPaginatedData(
+          rows,
+          state.pagination.perPage,
+          state.pagination.page
+        );
+      }
+
       return {
         ...state,
-        rows: rows,
+        rows,
```

Comment:
Why did this change from `rows: rows` to just `rows`?

Reply:
Just shorthand notation